ironfish/src/rpc/routes/wallet/importAccount.ts

```diff
@@ -31,7 +29,7 @@ export const ImportAccountRequestSchema: yup.ObjectSchema<ImportAccountRequest>
   .object({
     rescan: yup.boolean().optional().default(true),
     name: yup.string().optional(),
-    account: yup.mixed<RpcAccountImport | string>().defined(),
+    account: yup.string().defined(),
   })
```

Comment:
This is backwards incompatible, but I am guessing no one uses it. We should probably make sure the CLI isn’t using it anywhere

Reply:
We cannot use it because it's used in the account export flow. It actually represents the basic type of JSON encoding in the JSON encoder. I'm renaming the type in another PR and moving it into the exporter system to represent that.